README.md

```diff
@@ -0,0 +1,39 @@
+This is a browser-based tool used to assist in resolving duplicate urls in the PDAP Agencies Airtable.
+
+Pulls data from [PDAP Agencies Airtable](https://airtable.com/app473MWXVJVaD7Es/shr43ihbyM8DDkKx4/tblpnd3ei5SlibcCX).
+
+To utilize this tool, either utilize the PDAP Interface (not currently completed) or download a csv of the Agencies data and run initial_data_load.py to load the data into the database.
```

Comment:
what do you mean by "PDAP Interface" here?

Reply:
In that case I'm referring to the [PDAPInterface](https://github.com/maxachis/data-cleaning/blob/main/data_processing/PDAPInterface.py) class, which should streamline some of the functionality.

Reply:
Is this intended to be an interface for interacting with our own API, updating things and the like?